### PR TITLE
feat: let the user decide if they want planning to fail if no services are detected

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

**Comment by anwarchk** [comment]
If I understand it correctly, `move2kube` doesn't do anything with the bound services present in the cf manifest. So is this change from a transformers perspective ?

**Comment by ashokponkumar** [comment]
> If I understand it correctly, `move2kube` doesn't do anything with the bound services present in the cf manifest. So is this change from a transformers perspective ?

@anwarchk This particular PR has nothing to do with Cf bound services. But we are currently working on a capability to migrate bound services to either as Operators or services in Cloud catalogs. [Here](https://kubernetes.slack.com/archives/CR85S82A2/p1663827038349669) is a discussion on this topic. 

Please do let us know your usecase, how you are using Move2Kube, and how you would like Move2Kube transform the services, so that we can feed that into feature design for services migration. We can also continue the conversation in [slack](https://kubernetes.slack.com/archives/CR85S82A2).

File: cmd/plan.go [hunk 1]
```diff
@@ -139,10 +140,13 @@ func planHandler(cmd *cobra.Command, flags planFlags) {
 		logrus.Errorf("Unable to write plan file (%s) : %s", planfile, err)
 		return
 	}
+	logrus.Debugf("Plan : %+v", p)
 	logrus.Infof("Plan can be found at [%s].", planfile)
 	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
-		logrus.Debugf("Plan : %+v", p)
-		logrus.Fatalf("failed to find any services or default transformers.")
+		if flags.failOnEmptyPlan {
+			logrus.Fatalf("Did not detect any services in the directory %s", srcpath)
```
Comment (ashokponkumar) [inline cmd/plan.go:147]: we might also want to mention about default transformers similar to the previous message.

Comment (HarikrishnanBalagopal) [reply cmd/plan.go]: Done

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**